diemclient/client.go

```diff
@@ -276,9 +285,14 @@ func (c *client) GetEvents(key string, start uint64, limit uint64) ([]*Event, er
 	return ret, nil
 }
 
+// Submit hex-encoded signed transaction bytes to mempool.
+// This function ignores StaleResponseError and does not retry on any errors.
 func (c *client) Submit(data string) error {
-	ok, err := c.call(Submit, nil, data)
+	ok, err := c.callWithoutRetry(Submit, nil, data)
```

Comment:
is this changing the way the SDK works from a user perspective?

Reply:
no